crash-reporter/files/config.json

```diff
@@ -0,0 +1 @@
+{"port": 5004, "crash_dir": "./crashes/"}
```

Comment:
This will need a `"secret": "{{ pillar['crash-reporter']['secret'] }}"` entry as well.